src/pybind/mgr/volumes/fs/async_job.py

```diff
@@ -56,7 +53,12 @@ def run(self):
                 exc_type, exc_value, exc_traceback = sys.exc_info()
                 log.warning("traceback: {0}".format("".join(
                     traceback.format_exception(exc_type, exc_value, exc_traceback))))
-            time.sleep(1)
+            finally:
+                # when done, unregister the job
+                if vol_job:
+                    with self.async_job.lock:
+                        self.async_job.unregister_async_job(vol_job[0], vol_job[1], thread_id)
+                time.sleep(1)
```

Comment:
Why move the sleep into the `finally` clause?

Reply:
doesn't make much difference anyway, right? is the concern about code readability?

Reply:
Just stuck out to me.